src/opts.rs

```diff
@@ -42,6 +42,10 @@ pub enum Subcommand {
         /// The resulting file. Its extensions can be used to specify the compression formats.
         #[clap(required = true, value_hint = ValueHint::FilePath)]
         output: PathBuf,
+
+        /// If set, `ouch` will *not* compress hidden files or files ignored through .gitignore or .ignore
```

Comment:
This will appear in the `--help` message, so it should be a bit shorter, suggestion:

```rust
/// Ignore hidden files
```

I think that we should have another flag, `-i/--ignore` for `.gitignore` and `.ignore` files.